package.json

```diff
@@ -33,7 +33,7 @@
     "lint:package-lock": "node ./scripts/validate-package-lock.js",
     "servebuild": "serve -s build -l 3000",
     "test": "react-scripts test --testPathIgnorePatterns=./src/e2e",
-    "test:e2e": "react-scripts test ./src/e2e"
+    "test:e2e": "npm run servebuild & react-scripts test ./src/e2e; kill $!"
```

Comment:
Let's keep `test:e2e` as-is and create a new script `test:e2e:prod`. I'd like to still have an npm script for easily running the e2e tests against a running dev server.